src/lib/maps/Client.ts

```diff
@@ -65,7 +65,7 @@ export class Client {
           type: 'source',
           id: `${source}_${Date.now()}`,
           params: {
-            query: `SELECT * FROM ${source}`
+            query: type === 'sql' ? source : `SELECT * FROM ${source}`
```

Comment:
DIdn't we have this already?

Reply:
Well, we are using something similar at [Client.ts](https://github.com/CartoDB/web-sdk/blob/ad24d4ef51c5d4e4392d1c59e61800c486676ff3/src/lib/maps/Client.ts#L44) but with a MapOptions parameter